brianlong/www.validators.app
# 182958694 node pipeline

#### What's this PR do?
- create GossipNode and pipeline to fill the db

#### How should this be manually tested?
- run migrations
- run tests
- run script/update_gossip_nodes.rb
- check if GossipNodes are created
- check if associations are correct

#### What are the relevant tickets?
- https://www.pivotaltracker.com/story/show/182958694

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [Y] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
